lib/babel.js

```diff
@@ -3,6 +3,11 @@
 var debug = require('debug')('ava');
 
 if (debug.enabled) {
+	// Forward the `time-require` `--sorted` flag.
+	// Intended for internal optimization tests only.
+	if (exports.opts._sorted) {
```

Comment:
Should I completely drop the "time-require" module from package.json dependencies and the related `--sorted` stuff?